app/services/db/identity/unique_identities_per_ial_count.rb

```diff
@@ -0,0 +1,12 @@
+module Db
+  module Identity
+    class UniqueIdentitiesPerIalCount
+      def self.call(ial)
```

Comment:
ditto, let's inline this into the report class?

Reply:
why would we want to inline it?  it's reusable as it is